src/popup/locales/en.json

```diff
@@ -707,7 +707,7 @@
       "from": "From",
       "hash": "Transaction Hash",
       "home": "Home",
-      "scanAddress": "Scan AE Address",
+      "scanAddress": "Scan aeternity address",
```

Comment:
Here are all about aeternity, maybe better "account address"? Or meant that tokens can be sent to smart contract or oracle?

Reply:
> Here are all about aeternity, maybe better "account address"? Or meant that tokens can be sent to smart contract or oracle?

Yes, perhaps. 

To get the rationale behind you need to talk to the product & design people. Starting a discussion on figma might be the quickest way.

Reply:
The reason to change the text is that "Scan AE address" to "Scan aeternity address" was that "AE address" is no longer relevant with the introduction of different tokens. What @davidyuk suggested (Scan account address) sounds good to me but we may even replace it with just "Scan QR" as far as the modal opens in the context of account address input field on the "Send" screen.

Reply:
The popup is only shown when the wallet doesn't have permission to access the camera.

So it should be the following:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/17276821/123263709-db085b00-d52b-11eb-9f7e-19f9103ac559.png">

```suggestion
      "scanAddress": "Allow camera access to scan QR code",
```

Thanks!

Reply:
What about when access is allowed? Which title should we use?

![Screenshot 2021-06-25 at 10 04 15](https://user-images.githubusercontent.com/482351/123384324-ee074380-d59c-11eb-87a6-582dd63a1127.png)

Currently this is a single popup and changing title conditionally or splitting into two modals will require additional changes.